src/Codeception/Lib/Connector/Symfony.php

```diff
@@ -10,11 +10,26 @@ class Symfony extends \Symfony\Component\HttpKernel\Client
     private static $hasPerformedRequest;
 
     /**
+     * @var \Codeception\Module\Symfony
+     */
+    private $module;
+
+    /**
      * @var array
      */
     public $persistentServices = [];
 
     /**
+     * Symfony constructor.
+     * @param \Codeception\Module\Symfony $module
+     */
+    public function __construct($module)
+    {
+        $this->module = $module;
```

Comment:
I'd rather not have cyclic references here. Is that important for connector to be aware of module?